packages/editor/src/components/parts/program/event/ProgramStartPart.test.tsx

```diff
@@ -41,19 +48,15 @@ describe('StartPart', () => {
 
   test('configured', async () => {
     assertState('empty');
-    assertState('configured', {
-      javaClass: 'Bla'
-    });
+    assertState('configured', { javaClass: 'Bla', permission: { anonymous: true } });

```

Comment:
Do a separate check for the permission!
Imagine that your code which checks this state is wrong and the permission is not compared this would never fail because of the javaclass